### DISCUSSION TITLE
Go Conference mini 2023 Winter in Kyoto（仮） のキックオフMTG開催までに議論したいことをまとめる

### DISCUSSION THREAD

**Discussion by luccafort** [kyoto-go/go-conference-mini-2023#1] [General]
## Members

@kyoto-go/kyoto-go-organization-members 

@yebis0942 san says. https://gophers.slack.com/archives/CJG0WGYPL/p1682739790780739

```
Kyoto.goでもSendai.goさんみたいにオフラインでGo Conference mini的なことをやりたいね…！という相談をしていました。
まだGo Conferenceの運営スタッフの方にお話を通せてはいないですが、やるとしたらこんな感じかな、というのがまとまったのでご共有させてください！
●開催趣旨
・オフラインでGopherたちと話せる場を提供したい
・Go Conferenceに登壇できなかった方へ機会を提供したい
●開催時期
・11月下旬~12月上旬を予定
・8月開催がよさそう（お盆休暇に近い & 他のイベントと被らない）でしたが、逆算するとGo Conference終了後まもなくプロポーザルの募集を開始するスケジュール感になってやや余裕がなさそうなので冬としてます
●開催規模
・50名程度（Go Conference 2022 miniと同様）
```

**Comment by luccafort** [comment]
開催したいイベントの認識合わせがしたい。

> Kyoto.goでもSendai.goさんみたいにオフラインでGo Conference mini的なことをやりたい

上記だと解像度が低い状態なので、どういう条件があればSendai.goが行った Go Conference miniに近いになるのかの条件を洗い出したい。

**Reply by yebis0942** [reply]
https://scrapbox.io/kyotogo/Go_Conference_mini_2023_in_Kyoto#644c95312a5b840000531d27 で話していたのですが（Slackに貼り忘れていました :bow: ）、さらに掘り下げが必要な感はありますね。

何をしたいのかが明確になれば[なにができるイベントで、何ができないイベントなのか](https://github.com/kyoto-go/go-conference-mini-2023/discussions/1#discussioncomment-5760193)もそこから決まるはず。

**Reply by luccafort** [reply]
はっきりさせたい項目としては

- イベントの趣旨・目的はなにか？
- 誰がターゲットか？
- イベント開催後にどうなることを期待するか？
- イベントの成功条件・失敗条件はなにか？

かな。
いまはHowの話が多くて「こういうのできるといいよね」しかないのでどういうイベントにしたいのかが見えてこないなーというのが率直な意見です。
このあたり、一旦意見を発散させたいなのかすでにある程度イメージがあるので実現に向けて話し合いたいなのかでMTGの趣旨が変わると思います。

**Reply by luccafort** [reply]
> 何をしたいのかが明確になればhttps://github.com/kyoto-go/go-conference-mini-2023/discussions/1#discussioncomment-5760193もそこから決まるはず。

これはWantなのでそうではなくWhyが知りたいという意図の意見でした。

**Comment by luccafort** [comment]
多分一番大変なのは会場周り。
Sendai.goは元々コワーキングスペースをお借りしてイベントを開催していたりした。
Kyoto.goはそういったオフラインのナレッジや経験が不足しているのでどうしたいか？を確認したい。

**Reply by yebis0942** [reply]
「オフライン開催の経験がないので開催の段取りがうまくイメージできない」みたいな話は出ていました…

経験不足を埋める方法としては

* 小規模なオフラインイベントを開催して経験値を貯める
* 近隣のコミュニティ（Umeda.goさんなど）と協力して運営する
* すでに経験値のある他のコミュニティ（Kyoto.rbさん, Kyoto.jsさんなど）と合同イベントを開催してノウハウを学ばせてもらう

とかがあるのかなぁと思っています。

**Reply by luccafort** [reply]
そうですねー、経験しちゃう or 経験してる人をinviteするが多分一番手っ取り早いとは思ってました。
ただ、やりたいことによってオフライン開催に必要な経験値が異なるので、そこをどうするか？…という点で他コミュニティにヘルプをお願いしてもうまくいかない可能性があるのではないか？と少し懸念してます。

11〜12月頃の開催目処なのであれば少なくとも2度、理想的には3度くらい小規模でもいいので自分たちでオフラインイベントを企画、開催しておかないといきなり大規模なものはできないかなーと。

**Comment by luccafort** [comment]
なにができるイベントで、何ができないイベントなのかをはっきりさせたい。
どちらかというと **Why NOT** をはっきりさせたい。

ex. )
- スポンサーはできるのか
- ブースは出せるのか？
- 採用にまつわるアクションをおこなってもいいのか
- 発表は日本語でないといけないのか
- 開催はオンサイト、オンライン、ハイブリッド、その他のどれを期待するか
- イベント名をどうするか？（現在はイメージしやすいように仮称としてSendai.goのときのモデルを踏襲しています）

**Comment by luccafort** [comment]
## https://github.com/kyoto-go/go-conference-mini-2023/issues/6 で話したいこと

- このイベントが開催されるMUSTな条件はなにか？
- どうなればイベントは成功か。どうなればイベントは失敗か。

### 時間があれば話したいこと

- スポンサーを募集するのか？
- 会場をどこにするのか？
- (スポンサー募集するなら)ブースは出せるのか？
- スピーカーはどうやって集めるのか？

**Comment by yebis0942** [comment]
完了したので閉じます :pray:
